ui/appui-react/public/locales/en/UiFramework.json

```
@@ -280,7 +280,8 @@
     },
     "errorMessage": {
       "unknownError": "Something went wrong...",
-      "widgetPopoutFail": "Widget failed to popout, ensure you allow popout for this site."
+      "widgetPopoutFail": "Widget failed to popout, ensure you allow popout for this site.",
+      "popoutSafariFocusFail": "Popout focus restricted by Safari"
```

We should add the widget label in the message, so the user know what is happening.
```suggestion
      "popoutSafariFocusFail": "Popout widget '{{widgetLabel}}' attempted to get focus, but this action was restricted by the browser."
```

You can then fill this `{{widgetLabel}}` using a call similar to this:
```ts
UiFramework.localization.getLocalizedString(
  "widget.errorMessage.popoutSafariFocusFail",
  { widgetLabel: this.label, ns: UiFramework.localizationNamespace }
)
```